frontend/scss/organisms/_o-custom-tours-viewer.scss

```diff
@@ -0,0 +1,126 @@
+      left: 20px;
+      bottom: 20px;
+      margin-top: 20px;
+      margin-right: 20px;
+      color: $color__white;
+      font-family: $serif-font--loaded;
+      font-weight: 300;
+      font-size: 28px;
+      line-height: 1.14;
+      letter-spacing: -0.01em;
+    }
+
+    .aic-ct-grid-card__img {
+      aspect-ratio: 67/40;
+      height: auto;
+      width: 100%;
+    }
+
+    .default-img {
+      padding: 0 !important;
+      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='336' height='189' viewBox='0 0 336 189'%3E %3Cstyle type='text/css'%3E 	.st0{fill:%23333333;} 	.st1{fill:%23B7B1A5;} %3C/style%3E %3Crect x='-3' y='-2.5' class='st0' width='342' height='197'/%3E %3Cg%3E 	%3Cpath class='st1' d='M156.7,84.6c0.5-0.4,1-0.9,1.3-1.4c0.9-1.6,2.2-1.7,4.5-1.8c2.7-0.1,5.1-0.1,7.9,0c1.2,0,2.5,0.6,3,1.1 		c0.2,0.3,0.4,0.6,0.6,1c0.1,0.1,0.1,0.2,0.2,0.3c1.2,2,3.6,2,6.2,2c0.7,0,1.5,0,2.2,0c1.2,0.1,2.1,0.4,2.6,0.9 		c0.5,0.6,0.4,1.4,0.4,1.4l0,0.1v17.1c0,0.9-0.2,1.5-0.7,1.9c-0.3,0.2-0.6,0.4-0.9,0.5l1.6,1.3c0.2-0.1,0.4-0.3,0.7-0.5 		c0.9-0.8,1.3-1.9,1.3-3.3v-17c0-0.4,0.1-1.6-0.8-2.8c-0.8-1-2.2-1.6-4-1.6c-0.8,0-1.6,0-2.3,0c-2.5,0-3.9,0-4.5-1.1 		c-0.1-0.1-0.1-0.2-0.2-0.3c-0.2-0.4-0.5-0.8-0.7-1.2c-0.9-1.2-3-1.9-4.5-1.9c-2.8-0.1-5.2-0.1-8,0c-2.2,0.1-4.8,0.1-6.2,2.8 		c-0.3,0.5-0.6,0.8-1.1,1L156.7,84.6z'/%3E 	%3Cpath class='st1' d='M190.5,115.2l-6.4-5.5c0,0,0,0,0,0l-2-1.7h0l-8.5-7.3c0,0,0,0,0,0l-1.5-1.2c0,0,0,0,0,0l-10.3-8.8c0,0,0,0,0,0 		l-1.5-1.2c0,0,0,0,0,0l-4.9-4.2c0,0,0,0,0,0l-1.7-1.5c0,0,0,0,0,0l-13.1-11.1c-0.4-0.3-1-0.3-1.4,0.1c-0.4,0.4-0.3,1,0.1,1.3 		l11.5,9.8h-1.8c-0.4,0-1.9-0.1-3.1,0.7c-0.7,0.5-1.1,1.3-1.2,2.3l0,19.1l0,0.1c0.2,1.3,1.4,3.8,4.3,3.8h32.3l7.9,6.7 		c0.2,0.2,0.4,0.2,0.6,0.2c0.3,0,0.5-0.1,0.7-0.3C190.9,116.2,190.9,115.6,190.5,115.2z M160.5,92.2l10.3,8.7 		c-1.2,1-2.8,1.6-4.5,1.6c-3.8,0-6.9-3-6.9-6.7C159.4,94.5,159.8,93.2,160.5,92.2z M149.1,108.1c-1.8,0-2.3-1.7-2.4-2.2V87.1 		c0.1-0.4,0.2-0.7,0.5-0.9c0.5-0.4,1.4-0.4,1.7-0.3l0.1,0h3.5c0.2,0,0.4,0,0.6,0l6,5.1c-1,1.4-1.6,3.1-1.6,4.9c0,4.7,4,8.6,8.9,8.6 		c2.3,0,4.4-0.8,5.9-2.2l6.9,5.9H149.1z'/%3E 	%3Cpath class='st1' d='M174.2,99.6c0.6-1.1,0.9-2.4,0.9-3.8c0-4.7-4-8.6-8.9-8.6c-1.8,0-3.5,0.5-4.9,1.4l1.5,1.3 		c1-0.5,2.1-0.9,3.4-0.9c3.8,0,6.9,3,6.9,6.7c0,0.9-0.2,1.7-0.5,2.5L174.2,99.6z'/%3E %3C/g%3E %3C/svg%3E");
+    }
+
+    .aic-ct-grid-card__details {
+      width: 100%;
+      margin: auto;
+      display: inline-block;
+      padding: 16px 20px 24px;
+    }
+
+    .aic-ct-grid-card__artworks-count {
+      display: flex;
+      align-items: center;
+      margin-bottom: 16px;
+      gap: 8px;
+      font-family: $sans-serif-font--loaded;
+      font-size: 13px;
+      font-weight: 300;
+      line-height: 1.85;
+      letter-spacing: 0.01em;
+    }
+
+    .aic-ct-grid-card__text {
+      margin-bottom: 24px;
+      font-family: $serif-font--loaded;
+      font-size: 17px;
+      line-height: 1.41;
+      letter-spacing: -0.005em;
```

Comment:
See if we can replace this with a utility class

Reply:
Thanks, I've updated.